packages/itwin/measure-tools/src/ui-2.0/MeasureToolsUiProvider.tsx

```diff
@@ -24,6 +24,11 @@ import { UiFramework } from "@itwin/appui-react";
 
 export class MeasureToolsUiItemsProvider implements UiItemsProvider {
   public readonly id = "MeasureToolsUiItemsProvider";
+  private _props?: { itemPriority: number };
```

Comment:
could you pull this out and type this, and just reuse it?

Reply:
could you also update it to add groupPriority like Bill mentioned?

Reply:
> could you pull this out and type this, and just reuse it?

You mean create a new interface for it in somewhere like MeasureTools.ts?

Reply:
yes create a new interface/type, in this file would be ok i think